packages/modules/pt/src/utils/index.js

```diff
@@ -156,63 +157,256 @@ const getUsageType = (data) => {
   } else {
     return data?.usageCategoryMajor?.code;
   }
-}
+};
 
-export const setPropertyDetails = (data) => {
+const getOccupancyType = (data) => {
```

Comment:
add those values as a code itself , instead of this condition check for property type

do this way by setting as code 
 menu.push({ i18nKey: "COMMON_PROPTYPE_" + proptype[i].code.replaceAll(".", "_") ,code:proptype[i].code});
 
 referr this file packages\modules\pt\src\pageComponents\PropertyType.js